Source/Core/Core/IOS/USB/Bluetooth/BTEmu.cpp

```diff
@@ -412,48 +413,56 @@ bool BluetoothEmu::SendEventInquiryComplete()
 
 bool BluetoothEmu::SendEventInquiryResponse()
 {
-  static_assert(sizeof(SHCIEventInquiryResult) - 2 + (MAX_BBMOTES * sizeof(hci_inquiry_response)) <
+  // We only respond with the first discoverable remote.
+  // The Wii instructs users to press 1+2 in the desired play order.
+  // Responding with all remotes at once can place them in undesirable slots.
+  // Additional scans will connect each remote in the proper order.
+  constexpr u8 responses_per_result = 1;
+
+  static_assert(sizeof(SHCIEventInquiryResult) - 2 +
+                    (responses_per_result * sizeof(hci_inquiry_response)) <
                 256);
 
+  const auto iter = std::find_if(m_wiimotes.begin(), m_wiimotes.end(),
+                                 std::mem_fn(&WiimoteDevice::IsInquiryScanEnabled));
+  if (iter == m_wiimotes.end())
+  {
+    // No remotes are discoverable.
+    SendEventInquiryComplete(0);
+    return false;
+  }
+
+  const auto& wiimote = *iter;
+
   SQueuedEvent event(static_cast<u32>(sizeof(SHCIEventInquiryResult) +
-                                      m_wiimotes.size() * sizeof(hci_inquiry_response)),
+                                      responses_per_result * sizeof(hci_inquiry_response)),
                      0);
 
   SHCIEventInquiryResult* inquiry_result = (SHCIEventInquiryResult*)event.buffer;
-
   inquiry_result->EventType = HCI_EVENT_INQUIRY_RESULT;
-  inquiry_result->num_responses = 0;
+  inquiry_result->num_responses = 1;
```

Comment:
Should this be `responses_per_result` as well (even tho this is the same value)?